frontend/src/components/BottomSignUp/BottomSignUp.js

```diff
@@ -1,10 +1,16 @@
 import React, {Component } from 'react';
 import './BottomSignUp.css'
 import signupImage from "../../assets/soup-kitchen.jpg";
+import { withRouter } from 'react-router'
```

Comment:
I'd move this above the `import './BottomSignup.css'` line. Generally want to keep the library stuff at the top, then project-local stuff, then stuff specific to this component.